mhapsite/mhap/models.py

```diff
@@ -45,6 +45,7 @@ def get_list_url(self):
 #http://stackoverflow.com/questions/1737017/django-auto-now-and-auto-now-add
 class ChatMessages(models.Model):
     message = models.CharField(max_length=255)
+    message_type = models.CharField(max_length=255, default="default")
```

Comment:
Okay, then I could probably just use one character for the max length. And that's just in models.py, right? And then in the template I can just check if message_type is HELP or QUOTE?